src/in-mem/delay-response.ts

```diff
@@ -0,0 +1,30 @@
+import { Observable } from 'rxjs/Observable';
+
+/** adds specified delay (in ms) to both next and error channels of the response observable */
```

Comment:
please add docs you wrote for changelog also here so that nobody reading the code is wondering why we have a custom delay operator.

Reply:
Added reference to the CHANGELOG